lib/index.ts

```diff
@@ -3,26 +3,14 @@ import easySqs = require("./EasySqs");
 import errors = require("./CustomErrors");
 import interfaces = require("./Interfaces");
 
-export function createClient(awsConfig?: any) {
-
-  validateConfig(awsConfig);
+export function createClient(sqsConfig?: any, awsConfig?: any) {
```

Comment:
Could you please reverse the order of these parameters? awsConfig should remain the first parameter to avoid making a breaking change.